clang-tools-extra/clang-tidy/abseil/RedundantStrcatCallsCheck.cpp

```diff
@@ -22,7 +22,7 @@ namespace clang::tidy::abseil {
 //  - Make it work in macros if the outer and inner StrCats are both in the
 //    argument.
 
-void RedundantStrcatCallsCheck::registerMatchers(MatchFinder* Finder) {
+void RedundantStrcatCallsCheck::registerMatchers(MatchFinder *Finder) {
```

Comment:
Please leave out format-only changes from the PR. It makes the diff larger and it takes more effort to review it.

Reply:
Okay !!